docs/RestClients.md

```diff
@@ -41,6 +41,8 @@ Admin-on-rest ships 2 REST client by default:
 You can find REST clients for various backends in third-party repositories:
 
 * **[Epilogue](https://github.com/dchester/epilogue)**: [dunghuynh/aor-epilogue-client](https://github.com/dunghuynh/aor-epilogue-client)
+
+* **[DynamoDb](https://github.com/abiglobalhealth/aor-dynamodb-client)**: [abiglobalhealth/aor-dynamodb-client](https://github.com/abiglobalhealth/aor-dynamodb-client)
```

Comment:
Please add it in alphabetical order and don't add any empty line around it. Thanks!